source/features/highlight-deleted-and-added-files-in-diffs.tsx

```diff
@@ -61,11 +57,14 @@ async function init(): Promise<void | false> {
 	deinit.push(observer.abort);
 }
 
+function initCompare(): void {
+	onDiffFileLoad(init);
+}
```

Comment:
Maybe this could be replaced by `additionalListeners` + `additionalListenersOnly` in the `feature.add()`?

Reply:
Good to know that!